src/summary/history/allergy-card-level-two.component.tsx

```diff
@@ -0,0 +1,151 @@
+import React from "react";
+import { match } from "react-router";
+import { performPatientAllergySearch } from "./allergy-intolerance.resource";
+import { createErrorHandler } from "@openmrs/esm-error-handling";
+import styles from "./allergy-card-level-two.css";
+import SummaryCardRow from "../cards/summary-card-row.component";
+import SummaryCard from "../cards/summary-card.component";
+import dayjs from "dayjs";
+
+export function AllergyCardLevelTwo(props: AllergyCardLevelTwoProps) {
+  const [patientAllergy, setPatientAllergy] = React.useState(null);
+
+  React.useEffect(() => {
+    const abortController = new AbortController();
+
+    performPatientAllergySearch(
+      props.currentPatient.identifier[0].value,
+      abortController
+    )
+      .then(allergy => setPatientAllergy(allergy.data))
+      .catch(createErrorHandler());
+
+    return () => abortController.abort();
+  }, [props.currentPatient.identifier[0].value]);
+
+  function displayAllergy() {
+    return (
+      <SummaryCard
+        name="Allergy"
+        match={props.match}
+        styles={{ width: "100%" }}
+      >
+        <SummaryCardRow>
+          <table className={styles.allergyTable}>
+            <thead className="omrs-type-body-regular">
+              <tr>
+                <th>ALLERGEN</th>
+                <th>SEVERITY & REACTION</th>
+                <th>SINCE</th>
+                <th>UPDATED</th>
+              </tr>
+            </thead>
+            {patientAllergy &&
+              patientAllergy.entry.map(allergy => {
+                return (
+                  <tbody key={allergy.resource.id}>
+                    <tr>
+                      <td className={`omrs-bold`}>
+                        {allergy.resource.code.text}
+                      </td>
+                      <td
+                        className={`${styles.svgIcons} ${styles.capitalize} omrs-bold`}
+                      >
+                        <p>
+                          {allergy.resource.criticality === "high" ? (
```

Comment:
Shortcut for if statements in react, that makes it so you don't have to do `: null` after it. See https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

```suggestion
                          {allergy.resource.criticality === "high" && (
```

Reply:
Reverted to the previous state and worked on the requested changes